skypeweb/purplecompat.h

```diff
@@ -28,6 +28,12 @@
 //TODO remove this when dx adds this to the PurpleMessageFlags enum
 #define PURPLE_MESSAGE_REMOTE_SEND  0x10000
 
+static inline void purple_conversation_write_system_message_ts(PurpleConversation *conv, const gchar *msg, PurpleMessageFlags flags, time_t ts) {
```

Comment:
Thanks. It didn't occur to me that implementing two versions wasn't really necessary. Fixed.